legal-api/report-templates/template-parts/common/businessDetails.html

```diff
@@ -44,8 +44,8 @@
                  <div class="pt-2">
                      <span class="capitalize-text">{{business.state}}</span>
                      {% if business.state in ('HISTORICAL', 'LIQUIDATION') and (stateFilings or amalgamationOut) %}
```

Comment:
This `if` is not having any data for `LIQUIDATION`. You can change it only for `HISTORICAL`

Reply:
I trace the history of changes for this file, and think the logic was completely changed at this point: https://github.com/bcgov/lear/pull/3234/files#diff-afd71bbb4b60c26899a3b451c618bd22e79d50442876b69e6931321ff0dcd2f0
Historical firms won't have state filing info as suffix in business details any more. I don't find any new requirements.
I think we should keep the `LIQUIDATION` while the logic needs to be refined again. Can anyone help confirm this?